be/src/exec/partition/chunks_partitioner.h

```diff
@@ -182,7 +215,7 @@ class ChunksPartitioner {
         while (chunk_it != chunk_end) {
             // Because we first fetch chunks from hash_map, so the _partition_idx here
             // is already set to hash_map.size()
-            if (!consumer(_partition_idx, *chunk_it++)) {
```

Comment:
before: _partition_idx = sizeof(keys)
after: hash_map_with_key.kNullKeyPartitionIdx = 0
is that what you expected?

---

I see it

```cpp
            // partition_idx=0 is reserved by null key.
            auto next_partition_idx = hash_map.size() + 1;
```